src/cluster.c

```diff
@@ -190,6 +190,9 @@ int clusterLoadConfig(char *filename) {
          * base port. */
         n->cport = busp ? atoi(busp) : n->port + CLUSTER_PORT_INCR;
 
+        /* The plaintext port for client in a TLS cluster (n->pport) is not
```

Comment:
If we restart a node, that means we might show a pport of 0, since we haven't received their message yet. That seems incorrect.

Reply:
That's right (but arguably not worse than serving a TLS port to a plaintext client).

A future improvement is to add it to the nodes.conf file. I haven't looked into if/how it can be done in a backwards compatible way. Do you know?

Reply:
Extra fields can be added optionally, AFAIK. We do a splitargs on space and we don't explicitly validate the number, just that it's at least the number we expect. You're right this can be a future improvement thought.